enhancements/ingress/host-network-configurable-ports.md

```diff
@@ -182,7 +173,20 @@ This  proposal edits `IngressController.operator.openshift.io/v1`.
 
 ### Operational Aspects of API Extensions
 
-N/A.
+#### External load-balancer health checks setup
+
+An external load balancer can send HTTP probes to the ingress controller's StatsPort on a given node with the path /healthz/ready to determine whether the ingress controller is ready to receive traffic on the node. 
+Sending health check requests every 5 or 10 seconds, with a 5-second timeout and with a threshold of two successful or failed requests to become healthy or unhealthy, respectively, are well-tested values.
+For proper operation, the load balancer must not forward traffic to a node until /healthz/ready reports ready, and the load balancer must stop forwarding within a maximum of 45 seconds after /healthz/ready starts reporting not-ready. 
+
+#### Investigating issues
+
+Cluster administrators should watch out for the ingress controller status after updates:
+In case the router pod cannot be scheduled due to port unavailability (or any other scheduling concerns), the ingress controller's conditions will report the scheduling failure.
+In such a case, the cluster administrator might need to inspect other pods or daemonsets that use the same port. Router pod logs, OVN/SDN logs might be helpful to identify the root cause for scheduling failure.
```

Comment:
I would say "pods and system daemons"; a likely source of conflicts is running daemons on the host that aren't managed by OpenShift. `ss` or `netstat` would be an easy way to check for conflicts.

Reply:
We actually have a host-port registry -- https://github.com/openshift/enhancements/blob/master/dev-guide/host-port-registry.md -- as well as user-facing documentation (https://github.com/openshift/openshift-docs/blob/main/modules/installation-network-user-infra.adoc) for which ports we "claim". So end-user documentation should reference those ranges as being off-limits.